src/stylesheets/elements/_inputs.scss

```diff
@@ -1,7 +1,7 @@
 // scss-lint:disable QualifyingElement, PropertyCount
 
-$input-font-size-calc: scale($theme-font-family-input, $theme-font-size-base);
-$input-line-height-calc: lh($theme-font-family-input, $theme-line-height-input); // was 1.3
+$input-font-size-calc: scale($theme-font-family-forms, $theme-font-size-base);
+$input-line-height-calc: lh($theme-font-family-forms, $theme-line-height-input); // was 1.3
```

Comment:
I have been using verbose so it's clearer to people reading the code, let's save the shortened ones for prototyping.